### DISCUSSION TITLE
Empty Market function

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#513] [Ideas]
In the spirit of not wasting resources, would it be possible to add an "Empty Market" on the Special Orders, similar to Granary and Warehouse?
The market lady could take some boys and shuffle the goods over to a market trading in those goods.
Maybe for a touch of realism, some of the e.g. pottery doesn't make the trip because it broke on the way over.

**Comment by crudelios** [comment]
We're thinking about better ways to manage the markets, so this is a nice is a nice extra suggestion.
